migrations/000007_create_book_reviews.up.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE IF NOT EXISTS book_reviews(
+    book_id BIGINT NOT NULL REFERENCES books(id) ON DELETE CASCADE,
```

Comment:
It would be great to have bigint primary key here

Reply:
missed primary key statement, 
how about composite primary key? PRIMARY KEY (book_id, user_id)

Reply:
Sounds good if user can leave only one review.